internal/ingress/controller/controller.go

```diff
@@ -572,12 +572,6 @@ func (n *NGINXController) getBackendServers(ingresses []*extensions.Ingress) ([]
 		aUpstreams = append(aUpstreams, upstream)
 	}
 
-	if n.cfg.SortBackends {
```

Comment:
Maybe make the corresponding flag dummy and print a deprecation message?

Reply:
Scratch the deprecation part, I forgot that the flag also influenced the order of endpoints.